### DISCUSSION TITLE
3.2.0

### DISCUSSION THREAD

**Discussion by zsiki** [zsiki/GeoEasy#163] [General]
New point release to enhance internationalization. All message files in i18n folder are in UTF-8 code page. Special characters in file name are accepted (e.g. Cyrillic letters).
Units for angles and distances are set globally in Calculation parameters dialog, and it is used in field-book windows and calculation results.
Observations can be disabled selecting "Not used" type from the right button popup menu.
Some testings were made on OSX platform.

... and several bug fixed.

<hr /><em>This discussion was created from the release <a href='https://github.com/zsiki/GeoEasy/releases/tag/'>3.2.0</a>.</em>

**Comment by zvezdochiot** [comment]
>This discussion was created from the release 3.2.0.

But! 

https://github.com/zsiki/GeoEasy/releases

Last release: 3.1.3?!!!

**Comment by zvezdochiot** [comment]
Ok. But 3.1.3 could have been left. 

Stop! You only changed the title. The release remains the same. (now to add releases you drive in a new tag, after which you press "(+) add new tag". You need to return the old tag to its original state, and then add a new one). 

See also: https://github.com/Geo-Linux-Calculations/GeoEasy/releases

**Reply by zsiki** [reply]
I had trouble with tags and releases :(

**Reply by zvezdochiot** [reply]
Ok.
